.gitignore

```diff
@@ -2,5 +2,6 @@
 #  if you want to ignore files created by your editor/tools,
 #  please consider a global .gitignore https://help.github.com/articles/ignoring-files
 #  please do not open a pull request to add something created by your editor or tools
-dep
+cmd/dep/dep
+cmd/dep/dep.exe
```

Comment:
I'd like to see this change.  That 'dep' line in .gitignore causes Atom, at least, to disconnect its git montioring from the cmd/dep lib. I keep removing the line and being sure not to check in the change.